front-dex/src/app/components/pokemon-details/pokemon-details.component.html

```diff
@@ -0,0 +1,35 @@
+<main class="details-container">
+    <main class="details" [ngClass]="cardComponent.setBackgroundColor(pokemon.types, 'pokecard')">
```

Comment:
I'm counting 3 main tags. You can change the card i think to a div because its only styling?

Reply:
im sorry you can change this to an article

Reply:
Dat bepaal ik zelf wel

Reply:
perhaps